frontend/src/concepts/notebooks/getImageStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { NotebookImageAvailability } from '~/pages/projects/screens/detail/notebooks/const';
+import { NotebookImageData } from '~/pages/projects/screens/detail/notebooks/types';
+import { ImageStreamStatus } from './types';
+
+export const getImageStatus = (
```

Comment:
Can we instead just upgrade `NotebookImageAvailability` to include the updated state instead of having both NotebookImageAvailability and ImageStreamStatus

Reply:
id like to merge this function into getNotebookImageData becuase it is only ever used right next to it so makes sense to combine